cdap-docs/_common/common-build.sh

```diff
@@ -358,7 +377,63 @@ function display_version() {
   echo "PROJECT_SHORT_VERSION: ${PROJECT_SHORT_VERSION}"
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
-  echo ""
+  echo "GIT_BRANCH_PARENT: ${GIT_BRANCH_PARENT}"
+}
+
+function set_messages_file() {
+  TMP_MESSAGES_FILE="/tmp/$(basename $0).$$.tmp"
+  export TMP_MESSAGES_FILE
+}
+
+function cleanup_messages_file() {
+  if [[ "x${TMP_MESSAGES_FILE}" != "x" && -a ${TMP_MESSAGES_FILE} ]]; then
+    rm -f TMP_MESSAGES_FILE
+  fi
```

Comment:
Conditional is completely unnecessary as `rm -f` doesn't error on failure. Also, your variable isn't prepended with a `$` or in braces.